src/doctrines/Monoidal.jl

```diff
@@ -263,11 +267,10 @@ end
   # Adjoint mate of morphism f.
   mate(f::(A → B))::(dual(B) → dual(A)) ⊣ (A::Ob, B::Ob)
 
-  # Closed monoidal category
```

Comment:
In order for these axioms to make sense, we need corresponding term constructors.

Reply:
fixed